Source/Modules/go.cxx

```diff
@@ -6525,8 +6525,10 @@ class GO:public Language {
 	break;
       }
     }
-
-    if (Getattr(undefined_types, ty) && !Getattr(defined_types, ty)) {
+    
+    String* go_type = goType(n, ty);
+    
+    if (Getattr(undefined_types, go_type) && !Getattr(defined_types, go_type)) {
```

Comment:
As far as I can see, the index into `undefined_types` should be `ty` and the index into `defined_types` should be `go_type`.

Also you should `Delete(go_type)` somewhere.